creator/referral_tokens/models.py

```diff
@@ -0,0 +1,66 @@
+import uuid
+import pytz
+from datetime import datetime
+from django.db import models
+from django.utils import timezone
+from django.contrib.auth import get_user_model
+from django.conf import settings
+from creator.studies.models import Study
+from django.contrib.auth.models import Group
+
+
+User = get_user_model()
+
+
+class ReferralToken(models.Model):
```

Comment:
Can we add two more fields here?
- `claimed_by` - the user who claims the token
- `created_by` - the user who created the token
This will give us better tracking of how tokens are used in the case there are ever security questions we want to answer.